testdata/storage/misc/storageClass-with-stable-annotations.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: slow
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "false"
```

Comment:
```suggestion
```
It seems we could remove the annotations.

Reply:
> It seems we could remove the annotations.

Hi @Phaow After tries from my end. 

Testcase fails: 
    And admin creates new in-tree storageclass with:
      | ["metadata"]["annotations"]["storageclass.kubernetes.io/is-default-class"] | true |
if in-tree-storageClass-template.yaml
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: in-tree
 provisioner: kubernetes.io/#PROVISIONER#       

 With error: 
      undefined method `[]=' for nil:NilClass (NoMethodError)
      (eval):1:in `block (2 levels) in <top (required)>'

Job link:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/5224/console

Solutions:
1. Need to apply patch after creating the sc {"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}} 
2. Keep metadata,annotations,is default class in in-tree-storageclass-template.yaml and directly replace the paths which works fine without errors

Better to go with second option instead of applying patch after creating of sc.

Reply:
I'm okay to keep this annotations.

Reply:
@ropatil010 Got it! Thanks for the details!